docs/src/pages/landing/Themes.js

```diff
@@ -73,13 +49,13 @@ export default function Themes() {
               data-ga-event-action="click"
               data-ga-event-label="home"
               href="https://material-ui.com/store/?utm_source=docs&utm_medium=referral&utm_campaign=home-store"
-              className={classes.button}
+              sx={{ m: theme.spacing(4, 0, 6) }}
```

Comment:
Oh wow, the first time I see this pattern. Did you consider?

```suggestion
              sx={{ mt: 4, mb: 6 }}
```

Reply:
I just replaced them, didn't think much. But it's much more clear 👍